onboarding-enabler-java/src/main/java/org/zowe/apiml/eurekaservice/client/util/ApiMediationServiceConfigReader.java

```diff
@@ -196,7 +196,7 @@ private ApiMediationServiceConfig loadConfiguration(String internalConfigFileNam
         return serviceConfig;
     }
 
-    private void setServiceIpAddress(ApiMediationServiceConfig serviceConfig) throws ServiceDefinitionException {
+    public static void setServiceIpAddress(ApiMediationServiceConfig serviceConfig) throws ServiceDefinitionException {
```

Comment:
Could we rather move it to some utility class or something instead of making it public and static here?